Packs/IntegrationsAndIncidentsHealthCheck/IncidentFields/incidentfield-PlaybooksFailedCommands.json

```diff
@@ -17,9 +17,7 @@
  "neverSetAsRequired": false,
  "ownerOnly": false,
  "required": false,
- "selectValues": [
-  ","
- ],
```

Comment:
I see there was a value here, please consult with @altmannyarden before making this change and make sure it doesn't effect the use case

Reply:
Checked, works fine